packages/manager/modules/adp/src/deploy/security.html

```diff
@@ -0,0 +1,74 @@
+<!-- Security Section -->
```

Comment:
Could your template have at least a root node element (instead of multiple with a ng-if root nodes)?

Reply:
This is done